cmake_modules/BuildUtils.cmake

```diff
@@ -228,6 +228,7 @@ function(ADD_TERRIER_BENCHMARK REL_BENCHMARK_NAME)
     set(BENCHMARK_PATH "${EXECUTABLE_OUTPUT_PATH}/${BENCHMARK_NAME}")
     add_executable(${BENCHMARK_NAME} "${REL_BENCHMARK_NAME}.cpp")
     target_link_libraries(${BENCHMARK_NAME} ${TERRIER_BENCHMARK_LINK_LIBS})
+    set_target_properties(${BENCHMARK_NAME} PROPERTIES ENABLE_EXPORTS 1)
```

Comment:
Yes. We have to add this to expose the bytecode handler symbols to the benchmarks (`tpch_runner.cpp`). Otherwise we cannot execute TPL program from the benchmarks.